test.bash

```diff
@@ -24,6 +24,7 @@ int main() {
 EOF
 gcc -coverage -o foo foo.c
 ./foo
+export COVERALLS_REPO_TOKEN="fake testing token"
```

Comment:
Could you test whether the 'TRAVIS_JOB_ID` is set before set repo token?

We also want to test this logic on a travis machine.